demo/openapi.yaml

```diff
@@ -88,7 +88,7 @@ paths:
     parameters:
       - name: Accept-Language
         in: header
-        description: "The language you prefer for messages. Supported values are en-AU, en-CA, en-GB, en-US"
+        description: 'The language you prefer for messages. Supported values are en-AU, en-CA, en-GB, en-US'
```

Comment:
Looks like the pre-commit tooling changes the double-quotes to single-quotes automatically? I didn't make this (and the similar changes here directly)